projects/npm-tools/packages/npm-scripts/src/utils/createExportsBridges.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const {
+	addNamespace,
+	joinModuleName,
+	splitModuleName,
+} = require('@liferay/js-toolkit-core');
+const fs = require('fs');
+const path = require('path');
+const resolve = require('resolve');
+
+/* eslint-disable @liferay/no-dynamic-require */
```

Comment:
nit: I think this should technically be above the require statements. File-wide ignores should be right under the copyright

Reply:
done :+1: